<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingClass</class>
 <widget class="QMainWindow" name="ImageProcessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>300</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入影像路径：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输出影像路径：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="errMsgEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行信息：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>图像滤波</string>
     </property>
     <addaction name="actionMean_2"/>
     <addaction name="actionMid_2"/>
     <addaction name="actionGau_2"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>图像锐化</string>
     </property>
     <addaction name="actionRoberts_2"/>
     <addaction name="actionPrewitt_2"/>
     <addaction name="actionSobel_2"/>
     <addaction name="actionLaplacian_2"/>
     <addaction name="actionLoG"/>
     <addaction name="action_Laplacian"/>
    </widget>
    <addaction name="actionGF"/>
    <addaction name="actionBWM"/>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_fusion"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGF">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>高斯滤波</string>
   </property>
  </action>
  <action name="actionBWM">
   <property name="text">
    <string>图像盲水印</string>
   </property>
  </action>
  <action name="actionMean">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="actionMid">
   <property name="text">
    <string>中值滤波</string>
   </property>
  </action>
  <action name="actionGau">
   <property name="text">
    <string>高斯滤波</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionMean_2">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="actionMid_2">
   <property name="text">
    <string>中值滤波</string>
   </property>
  </action>
  <action name="actionGau_2">
   <property name="text">
    <string>高斯滤波</string>
   </property>
  </action>
  <action name="actionRoberts_2">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionPrewitt_2">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionSobel_2">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionLaplacian_2">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionLoG">
   <property name="text">
    <string>LoG边缘监测</string>
   </property>
  </action>
  <action name="action_Laplacian">
   <property name="text">
    <string>更厉害的Laplacian滤波</string>
   </property>
  </action>
  <action name="action_fusion">
   <property name="text">
    <string>图像融合</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageProcessing.qrc"/>
 </resources>
 <connections/>
</ui>
